kuma/static/styles/components/wiki/interactive.scss

```diff
@@ -12,6 +11,12 @@
     height: 665px;
 }
 
+.interactive-editor-survey {
+    margin: 0 auto;
```

Comment:
We should add the `margin-bottom: $grid-spacing` here to keep space between the editor and the next page elements.